### PR TITLE
[SlackTextViewController] Updated to v1.9.5

### PR DESCRIPTION
**Pull request by mattleibow**

### CODE REVIEW

File: iOS/SlackTextViewController/License.md [hunk 1]
```diff
@@ -14,4 +10,4 @@ The above copyright notice and this permission notice shall be included in all c
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-20160601
+20161201
```
Comment (bholmes) [inline iOS/SlackTextViewController/License.md]: This is a date.  change to 20170207

File: iOS/SlackTextViewController/External-Dependency-Info.txt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Xamarin Components for iRate Rating Prompt incorporates third party 
+material from the projects listed below. The original copyright notice and 
+the license under which Microsoft received such third party material are set 
+forth below.  Microsoft reserves all other rights not expressly granted, 
+whether by implication, estoppel or otherwise.
+
+########################################
+# SlackTextViewController
+# https://github.com/slackhq/SlackTextViewController
```
Comment (bholmes) [inline iOS/SlackTextViewController/External-Dependency-Info.txt:12]: Please check the repo for " Copyright (c) 2013 Lukas Kubanek"

If we are using code under that copyright please add it to this file.

File: iOS/SlackTextViewController/nuget/SlackTextViewController.nuspec [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.SlackTextViewController</id>
+    <title>SlackTextViewController for Xamarin.iOS</title>
```
Comment (bholmes) [inline iOS/SlackTextViewController/nuget/SlackTextViewController.nuspec:5]: No icon listed.  Can you point to the raw link in Github for that?

**Review by bholmes — CHANGES_REQUESTED** [review]
I did not do a test build.  Please perform a rebuild locally before merge.

File: iOS/SlackTextViewController/External-Dependency-Info.txt (hunk 2 above)
Comment (mattleibow) [reply iOS/SlackTextViewController/External-Dependency-Info.txt:12]: That is just for the sample, and I don't use it in ours.

**Review by bholmes — APPROVED** [review]

**Pull request merged**